stubs/cachetools/cachetools/__init__.pyi

```diff
@@ -99,14 +97,26 @@ class TLRUCache(_TimedCache[_KT, _VT]):
     def ttu(self) -> Callable[[_KT, _VT, float], float]: ...
     def expire(self, time: float | None = None) -> list[tuple[_KT, _VT]]: ...
 
+@overload
+def cached(
+    cache: MutableMapping[_KT, Any] | None,
+    key: Callable[..., _KT] = ...,
+    lock: AbstractContextManager[Any] | None = None,
+    condition: Condition | None = None,
+    info: bool = False,
+) -> IdentityFunction: ...
+@overload
+@deprecated("Passing `info` as positional parameter is deprecated.")
 def cached(
     cache: MutableMapping[_KT, Any] | None,
     key: Callable[..., _KT] = ...,
     lock: AbstractContextManager[Any] | None = None,
+    condition: bool | None = None,
     info: bool = False,
```

Comment:
We should remove the `info` argument in the second overload. Passing both `condition` (as bool) and `info` will not do whatever the user expects and is very likely a bug.